src/modules/common/Logger.ts

```diff
@@ -75,25 +81,51 @@ export class Logger {
      * @return A transport that can be passed to `logger.add`
      */
     public static defaultDatabaseTransport(mongodb_url: string) {
+        const customLevel: config.AbstractConfigSetLevels =
+        {
+            http: 0,
+            info: 1,
+            error: 2,
+            debug: 3,
+            warn: 4,
+        };
         const options = {
-            level: "http",
+            level: 'info',
+            db: mongodb_url,
+            collection: 'operation_logs',
+            tryReconnect: true,
+            format: combine(
+                timestamp(),
+                json(),
+                metadata({ fillExcept: ['message', 'level', 'timestamp', 'label'] }),
+            ),
+            options: { useUnifiedTopology: true }
+        };
+        const access_options = {
+            level: 'http',
             db: mongodb_url,
-            collection: "stoa_logs",
+            collection: 'access_logs',
             tryReconnect: true,
             format: combine(timestamp(), json(), metadata({ fillExcept: ["message", "level", "timestamp", "label"] })),
             options: { useUnifiedTopology: true },
         };
-        return new MongoDB(options);
+        return winston.createLogger({
+            levels: customLevel,
+            transports: [
+                new MongoDB(options),
+                new MongoDB(access_options),
```

Comment:
It must determine whether to use mongoDB it by config settings.

Reply:
we are just using mongo url from config, rest of the configuration is for database transport logger, creating multiple logger transport with different logger level

Reply:
`defaultDatabaseTransport` It is not being used in the code.
I'd like to decide whether to use it by config.logging setting.